backend/src/Core/FurnitureShop.Core.Services/CQRS/Mobile/Favourites/MyFavouritesQH.cs

```diff
@@ -66,19 +65,18 @@ private async Task<List<Guid>> GetProductsInShoppingCart(CoreContext context)
             var shoppingCartId = shoppingCart.Id;
 
             return await dbContext.ShoppingCartProduct
-                .Where(shp => shp.ShoppingCartId == shoppingCartId.Value && shp.ProductId != null)
+                .Where(shp => shp.ShoppingCartId == shoppingCartId.Value)
                 .Select(shp => shp.ProductId!.Value).ToListAsync();
         }
 
         private async Task<List<Guid>> GetProductsInFavourites(CoreContext context)
         {
             return await dbContext.Favourites
-                .Where(f => f.UserId == context.UserId.Value && f.ProductId != null)
+                .Where(f => f.UserId == context.UserId.Value)
                 .Select(f => f.ProductId!.Value).ToListAsync();
```

Comment:
Why it is nullable

Reply:
po zmianach w enitity framwork już nie jest nullable